### PR TITLE
Replacing relative reference in excel formula and add completer

### LINKED ISSUE #54: Change the relative reference used in budget formatting function to absolute reference

### LINKED ISSUE #55: Use completer to turn the callback function of Excel and Office into future

### PR DESCRIPTION
**Pull request by hanj7221**
closes #54 and closes #55

### CODE REVIEW

File: lib/src/excel.dart [hunk 1]
```diff
@@ -167,12 +167,13 @@ class ExcelDart {
...
 
-      // Sets the sheet as active and syncs context.
+      // Sets the sheet as active.
       sheet.activate();
-      context.sync();
 
       // All operations are complete.
       populateCompleter.complete();
+
+      return context.sync();
```
Comment (hanj7221) [inline lib/src/excel.dart:176]: Adding back the return statement....
Excel js expects this callback function to return context.sync(). Even though the population still works without it, Excel js will complain in the console.

File: lib/src/query_component.dart [hunk 2]
```diff
@@ -77,17 +80,17 @@ class QueryComponent {
...
+              json.stringify(jsonGetQueryResponse));
 
       // Downloads the report and parse the response into a revenue map.
       final report =
-      await _queryService.execReportingDownload(reportingDownloadPath);
+          await _queryService.execReportingDownload(reportingDownloadPath);
 
       return ReportingQueryParser.parseRevenueFromJsonString(report);
-    } catch(e) {
+    } catch (e) {
```
Comment (ruben-aghayan) [inline lib/src/query_component.dart:93]: has it been throwing? why add this now if not adding handling?

Comment (hanj7221) [reply lib/src/query_component.dart:93]: The try catch was added in a previous [pr](https://github.com/googleinterns/dv360-excel-plugin/pull/50#discussion_r453093375). I guess I forget to run dartfmt when I merge it, so the indentation was wrong T.T

Comment (ruben-aghayan) [reply lib/src/query_component.dart:93]: yea but I only just noticed it.
lets merge this for now but think about why this is here. if we are encountering this error then lets figure out why and if we arent we should let it throw so we arent swallowing errors we dont know about.

**Review by ruben-aghayan — APPROVED** [review]

**Pull request merged**